docs/source/quickstart/index.rst

```diff
@@ -168,30 +168,36 @@ Editor Integration
 If your preferred editor supports language servers, you can configure
 it to provide completion and validation while authoring policies.
 
-First generate use custodian to generate a jsonschema file::
+First generate use custodian to generate a json schema file::
 
   custodian schema --json > schema.json
 
-Next install
-``yaml-language-server <https://github.com/redhat-developer/yaml-language-server>``
-into your editor of choice.
+Next install a YAML plug-in for your editor, like `YAML for Visual Studio Code
+<https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml>_` or
+`coc-yaml for coc.nvim <https://github.com/neoclide/coc-yaml>_`. Both plug-ins
+use the `yaml-language-server
+<https://github.com/redhat-developer/yaml-language-server>_` under the hood.
 
-You'll then configure it to use the generated `schema.json` as the
-schema for your policy files. For example in visual studio code
-settings that looks like::
+You'll then need to configure your plug-in to use the generated `schema.json`
+as the schema for your policy files. For example in Visual Studio Code,
+navigate to the settings for the YAML plug-in and under Schemas, edit
+configuration file and add the following schema configuration::
 
       "yaml.schemas": {
         "./schema.json": "*yml"
       },
```

Comment:
Could we put a note here about doing ?

"/Users/username/schema.json": "*yml"
vs 
"file:///Users/username/schema.json": "*yml"

that drove me crazy yesterday

Reply:
I actually use the full path in my `settings.json`:

```
/absolute/path/to/schema.json": "*yml"
```

What should the note say about file:// ?

Reply:
it probably shouldn't mention it all since thats broken

Reply:
I filed an issue so hopefully they fix it soon. Maybe it actually is best not to mention since they may update and then we would need to update the docs again later.

https://github.com/redhat-developer/vscode-yaml/issues/291

Reply:
Ah, thanks for explaining that, I didn't even see that in the vscode-yaml documentation. Ok cool, looks like this conversation can be resolved.